src/utils.ts

```diff
@@ -25,6 +36,14 @@ export function parseUrl(url: string): Url {
   return urlObject;
 }
 
+export function posixify(pathname: string): string {
+  return pathname.replace(/\\/g, '/');
```

Comment:
Apparently they are valid in posix-land but not Windows-land.  For this reason, I've made the path transform functions no-ops in posix-land.